### ISSUE TITLE
Missed a few steps...?

### ISSUE THREAD

**Issue by terentz** [jagenjo/litefilesystem.js#3]
I'm in the process of installing WebGLStudio at home, and although they're no difficulty in extrapolating for some of us, many may be somewhat lost at your omission of the MySQL steps necessary in the installation... :/

**Comment by jagenjo** [comment]
You are right, I wanted to create a script to install everything but never found time. I was expecting somebody could help me as I am not used to create deploy scripts.
Did you found any problem?

**Comment by coderofsalvation** [comment]
I'm working on a dockerscript..will share asap when done
